RevenueCat/Plugins/Android/PurchasesWrapper.java

```diff
@@ -292,6 +292,10 @@ public static boolean isAnonymous() {
         return CommonKt.isAnonymous();
     }
 
+    public static boolean isConfigured() {
+        return Purchases.isConfigured();
```

Comment:
I guess we bypass PHC on this one... But looks like we also do on the other hybrids. It's probably ok.